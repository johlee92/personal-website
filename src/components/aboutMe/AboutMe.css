.about-me {
    width: 100%;
    display: flex;
    flex-direction: column;
    align-items: center;
}

.about-me-image {
    margin: 10px 0;
    width: 80%;
    max-width: 500px;
}

.about-me-message {
    margin: 10px 0;
    width: 80%;
    max-width: 500px;
}

@media (min-width: 720px) {
    .about-me {
        height: 100%;
        justify-content: center;
    }
}