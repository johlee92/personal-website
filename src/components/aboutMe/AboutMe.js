import React, { Component } from 'react';
import './AboutMe.css';

class AboutMe extends Component {
    render() {
        return (
            <div>
            </div>
        )
    }
}

export default AboutMe;