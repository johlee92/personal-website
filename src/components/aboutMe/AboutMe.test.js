import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import AboutMe from './AboutMe';

describe(`AboutMe component`, () => {
    it('renders without crashing', () => {
        const div = document.createElement('div');
        ReactDOM.render(
                <AboutMe />
            , div);
        ReactDOM.unmountComponentAtNode(div);
    });

    it('renders the component as expected', () => {
        const tree = renderer
            .create(
                <AboutMe />
            )
            .toJSON();
        expect(tree).toMatchSnapshot();  
    });
})
