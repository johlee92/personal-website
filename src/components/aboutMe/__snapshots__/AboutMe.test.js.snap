// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutMe component renders the component as expected 1`] = `<div />`;
